joomla/joomla-cms
# [4.2] Add missing properties

Pull Request for Issue # .

### Summary of Changes
This simple PR adds missing properties to various classes in our libraries to avoid deprecated warning missing properties in PHP 8.2. These missing properties are detected automatically by rector [CompleteDynamicPropertiesRector](https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#completedynamicpropertiesrector) rule.

### Testing Instructions
1. Code review should be enough
2. Or apply patch, installing extension and make sure it is still working OK.


### Actual result BEFORE applying this Pull Request
Deprecated warnings when installing package extensions on PHP 8.2 and in some other places.


### Expected result AFTER applying this Pull Request
No warnings anymore.

### Link to documentations
